libsolidity/parsing/ParserBase.h

```diff
@@ -23,6 +23,7 @@
 #pragma once
 
 #include <memory>
+#include <libsolidity/interface/ErrorReporter.h>
 #include <libsolidity/interface/Exceptions.h>
```

Comment:
The `Exceptions.h` include, since in other files the `ErrorReporter.h` superseded the exceptions.